projects/Mallard/src/hooks/use-issue-summary.tsx

```diff
@@ -8,12 +7,15 @@ import gql from 'graphql-tag'
 import { getSetting } from '../helpers/settings'
 import { useQuery } from './apollo'
 import ApolloClient from 'apollo-client'
+import { Dispatch } from 'react'
 
 interface IssueSummaryState {
+    __typename: 'IssueSummary'
```

Comment:
Is the typename needed here given your more recent PR to remove typename from the Weather data models?

Reply:
Yes it is needed afaik., that a top-level, queryable Apollo object. So it needs a type in the Apollo model. The ones are removed are considered plain "JSON" data, and fields are not queryable separately (so no granularity for re-render). But here, the granularity is useful.